Assets/Resources/BillingMode.json

```diff
@@ -0,0 +1 @@
+{"androidStore":"GooglePlay"}
```

Comment:
I guess this file is unnecessary.

Reply:
I agree. Maybe the `Resources/*` files are unnecessary 🤔 I'll remove them and add the path to `.gitignore`